spec/features/goal_tool_spec.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   scenario "Displays due date" do
-    expect(page).to have_content "Due: #{Date.today.to_s(:date)}"
+    expect(page).to have_content "Due: #{Date.today.strftime("%b %d %Y")}"
```

Comment:
I would point out that this is a feature spec, and as such it has a different purpose than a unit spec. I would argue that it should be more lenient and you could even get away with just testing for `"Due: "` without a timestamp. Unless the timestamp format makes or breaks the feature, don't worry about it in the feature spec. But maybe I'm alone in this.